crates/js-sys/src/lib.rs

```diff
@@ -314,6 +314,13 @@ extern "C" {
     #[wasm_bindgen(method)]
     pub fn sort(this: &Array) -> Array;
 
+    /// The splice() method changes the contents of an array by removing existing elements and/or
+    /// adding new elements.
+    ///
+    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
+    #[wasm_bindgen(method)]
+    pub fn splice(this: &Array, start: u32, delete_count: u32, items: &JsValue) -> Array;
```

Comment:
I think the argument `items` here should be `item`, right? In that if an array is passed it's not treated as varargs, just as an array in an array?

Reply:
totally. tried to treat it as an array at first and forgot to change the name.